### ISSUE TITLE
Package epadd-settings with epadd-standalone.jar epadd.exe and epadd.dmg

### ISSUE THREAD

**Issue by hangal** [ePADD/epadd#151]
epadd-settings not being in the right place is a common failure mode.
Would be nice to have it packaged all together.

**Comment by hangal** [comment]
We can have versioning for epadd-settings just like for serialized archive. (so old versions of epadd can continue to run).

From now on, we can have ~/epadd-settings/v2, etc.
The standalone/exe/dmg will copy these files from the standalone.jar if the target dir doesn't already exist before starting tomcat.

**Comment by peterchanws** [comment]
Can we allocate 32 GB ram as default?

**Comment by peterchanws** [comment]
Please remove the log in the settings files we bundle with the program.
